docs/infra/quickstart.md

```diff
@@ -0,0 +1,86 @@
+---
+head:
+  - - meta
+    - name: "twitter:title"
+      content: Quickstart | zkSync Docs
+---
+
+# Quick Start Guide for zkSync Node
+
+## Prerequisites
+
+- **Installations Required:**
+  - [Docker Compose](https://docs.docker.com/compose/install/)
+  - [Docker](https://docs.docker.com/get-docker/)
+
+## Running a zkSync Node Locally
+
+### Starting the Node
+
+- **For a Mainnet instance:**
+
+  ```sh
+  cd docker-compose-examples
+  docker compose --file mainnet-external-node-docker-compose.yml up
+  ```
+
+- **For a Testnet instance:**
+
+  ```sh
+  cd docker-compose-examples
+  docker compose --file testnet-external-node-docker-compose.yml up
+  ```
+
+### Resetting the Node State
+
+- **For a Mainnet instance:**
+
+  ```sh
+  cd docker-compose-examples
+  docker compose --file mainnet-external-node-docker-compose.yml down --volumes
+  ```
+
+- **For a Testnet instance:**
+
+  ```sh
+  cd docker-compose-examples
+  docker compose --file testnet-external-node-docker-compose.yml down --volumes
+  ```
+
+- **To reset its state, run:**
+
+```sh
+cd docker-compose-examples
+docker compose --file testnet-external-node-docker-compose.yml down --volumes
+```
+
+### Monitoring Node Status
+
+Access the local Grafana dashboard to see the node status after recovery:
+[Local Grafana Dashboard](http://localhost:3000/d/0/external-node).
+
+### API Access
+
+- **HTTP JSON-RPC API:** Port `3060`
+- **WebSocket API:** Port `3061`
+
+### Important Notes
+
+- **Initial Recovery:** The node will recover from a snapshot on its first run, which may take up to 10 hours. During this period, the API server will not serve any requests.
+- **Historical Data:** For access to historical transaction data, consider recovery from DB dumps. Refer to the Advanced Setup section for more details.
+- **DB Dump:** For nodes that operate from a DB dump, which allows starting a zkSync node with a full historical transactions history, refer to the documentation on running from DB dumps at [03_running.md](https://github.com/matter-labs/zksync-era/blob/main/docs/guides/external-node/03_running.md).
```

Comment:
Are we still calling this directory `external-node` after the latest updates? (I'm not sure either, but worth asking)
```suggestion
- **DB Dump:** For nodes that operate from a DB dump, which allows starting a zkSync node with a full historical transactions history, refer to the documentation on running from DB dumps at [03_running.md](https://github.com/matter-labs/zksync-era/blob/main/docs/guides/zksync-node/03_running.md).
```

Reply:
Yes, as we will have a larger config effort happening in the future with breaking changes and we'll update these then as well, more info in Slack